packages/front-end/components/Metrics/MetricsList.tsx

```diff
@@ -323,7 +325,7 @@ const MetricsList = (): React.ReactElement => {
         return item.type;
       },
       tag: (item) => item.tags,
-      project: (item) => item.projects,
+      project: (item) => item.projectNames,
```

Comment:
Should we concat these arrays so you can search by either id or name?  Main use case for searching by id is constructing a direct link to a filtered view like `?q=project:prj_abc123`.  Much more natural to use ids for this